fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -192,21 +194,45 @@ public Response options() {
      * Retrieve the node profile
      *
      * @param rangeValue the range value
-     * @return triples for the specified node
+     * @return the binary or the triples for the specified node
      * @throws IOException if IO exception occurred
      */
     @GET
-    @Produces({TURTLE + ";qs=10", JSON_LD + ";qs=8",
-            N3, N3_ALT2, RDF_XML, NTRIPLES, APPLICATION_XML, TEXT_PLAIN, TURTLE_X,
-            TEXT_HTML, APPLICATION_XHTML_XML, "*/*"})
+    @Produces({TURTLE + ";qs=10", "*/*"})
     public Response describe(@HeaderParam("Range") final String rangeValue) throws IOException {
         checkCacheControlHeaders(request, servletResponse, resource(), session);
 
         LOGGER.info("GET resource '{}'", externalPath);
+
+        final ImmutableList<MediaType> acceptableMediaTypes =
+                new ImmutableList.Builder<MediaType>().addAll(headers.getAcceptableMediaTypes()).build();
+
+        if (acceptableMediaTypes.size() > 0) {
+
+            final List<Variant> possibleVariants = new ArrayList();
+            if (resource() instanceof FedoraBinary) {
+                final String lang = null;
+                final String enc = null;
+                possibleVariants.add(new Variant(MediaType.valueOf(((FedoraBinary) resource()).getMimeType()),
+                        lang, enc));
+            } else {
+                possibleVariants.addAll(RDFMediaType.POSSIBLE_RDF_VARIANTS);
+            }
+
+            final boolean match = acceptableMediaTypes.stream().anyMatch(x ->
+                    x.isWildcardType() || possibleVariants.stream().anyMatch(t -> t.getMediaType().isCompatible(x)));
+
+            if (!match) {
+                 LOGGER.info("Unable to produce content in the requested mime-type(s): " );
+                 acceptableMediaTypes.stream().forEach(x -> LOGGER.info(x.toString()));
```

Comment:
Actually, it doesn't appear as if I can do this here, since I'd be using an instanceMethod that doesn't take a MediaType as it's argument.  
`acceptableMediaTypes.stream().forEach(LOGGER::info)` equals, I think, 
`acceptableMediaTypes.stream().forEach(x -> LOGGER::info(x))`  <-- x is a MediaType; won't compile
Seems that I really need the `x.toString()` as the argument.  Thoughts?